arithmetic_analysis/bisection.py

```diff
@@ -23,24 +23,25 @@ def bisection(function: Callable[[float], float], a: float, b: float) -> float:
     end: float = b
     if function(a) == 0:  # one of the a or b is a root for the function
         return a
-    elif function(b) == 0:
+    if function(b) == 0:
         return b
-    elif (
+    if (
         function(a) * function(b) > 0
     ):  # if none of these are root and they are both positive or negative,
         # then this algorithm can't find the root
```

Comment:
Can this fix be done with a `ruff --fix` rule?

Please put these comments on a separate line before the `if`.